_data/README.md

```diff
@@ -0,0 +1,14 @@
+# Projektdaten
+
+Projektdaten werden nach diesem Schema erfasst:
+
+```
+'search_it': {'lead': 'tyrant88', 'transferred_at': '2017-02-05'}
+```
+
+* {Key}  
+Entspricht dem Namen des Repos.
+* __`lead`__  
```

Comment:
Können wir gerne machen, nur sind die Möglichkeiten in Yaml ja eher begrenzt. Ich fand README praktisch, weil man den Kram dann sehr gut auch direkt in der GitHub-Weboberfläche sieht.